test/src/test/java/com/distkv/client/IntProxyTest.java

```diff
@@ -37,12 +38,23 @@ public void testKeyNotFoundWhenGetting() throws InvalidProtocolBufferException {
   }
 
   @Test
-  public void testExpireList() throws InterruptedException, InvalidProtocolBufferException {
+  public void testExpireList() {
     DistkvClient client = newDistkvClient();
     client.ints().put("k1", 1);
-    client.ints().expire("k1", 1);
-    Thread.sleep(3000);
-    Assert.assertThrows(KeyNotFoundException.class, () -> client.ints().get("k1"));
+    client.ints().expire("k1", 1000);
+    boolean result = RuntimeUtil.waitForCondition(() -> {
+      try {
+        try {
+          client.ints().get("k1");
+        } catch (InvalidProtocolBufferException e) {
+          //ignore
```

Comment:
Is swallowing exception a  bad practice?

Reply:
Why not 
```java
try {

} catch (KeyNotFoundException) {

} catch (InvalidProtocolBufferException) {

}
```